src/Services/Messages/DispatchMessage.php

```diff
@@ -62,12 +70,25 @@ public function handle(Message $message): ?string
         return $messageId;
     }
 
+    /**
+     * The message's subject is merged and persisted to the database
+     * so that we have a permanent record of the merged tags at the
+     * time of dispatch.
+     */
+    protected function mergeSubject(Message $message): Message
```

Comment:
Why do we persist the subject but explicitly do not persist the content as merged? Is it just because it is lighter weight to do so?